docs/_source/guides/llms/practical_guides/collect_responses.ipynb

```diff
@@ -24,7 +31,12 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "If your dataset doesn't have any annotation overlap i.e., all records have one response only, the post-processing stage will be quite simple. You just need to transform the dataset into whatever format you need for training."
+    "If your dataset doesn't have any annotation overlap i.e., all records have at most one response, the post-processing stage will be quite simple, since you won't need to decide which annotations to keep and which to discard.\n",
```

Comment:
```suggestion
    "If your dataset doesn't have any annotation overlap i.e., all records have at most one response, the post-processing stage will be quite simple since you won't need to decide which annotations to keep and which to discard.\n",
```